frontend/src/screens/learner.tsx

```diff
@@ -71,57 +70,99 @@ const CuratedStudies: FC = () => {
 
 const LearnerDashboard = () => {
     const env = useEnvironment()
+    const [scroll, scrollTo] = useWindowScroll();
```

Comment:
This is making the page slow and laggy, when other elements are clicked which do not utilize this component. I think It can be simplied by using `window.scroll(0, 0)` instead of state management.